lib/compress.js

```diff
@@ -1053,9 +1053,7 @@ merge(Compressor.prototype, {
             var val = {};
             for (var i = this.properties.length; --i >= 0;) {
                 var prop = this.properties[i];
-                if (prop instanceof AST_ObjectKeyVal) {
-                    val[prop.key] = ev(prop.value, compressor);
-                }
+                val[prop.key] = ev(prop.value, compressor);
```

Comment:
I think that's the correct way to do it for ES5/master.

Can you please add a comment above this live that harmony/ES6 will additionally have to run ev() on prop.key which will throw on side effects?